.eslintrc.cjs

```diff
@@ -55,7 +55,7 @@ module.exports = {
 
         // JSDoc @param types are mandatory for JavaScript
         'jsdoc/require-param-description': 'off',
-        'jsdoc/require-param-type': 'error',
+        'jsdoc/require-param-type': 'off',
```

Comment:
this is actually really helpful. If you add a param, this makes sure you type it. If you don't add a param it doesn't error. I'd personally keep this as `error`